### ISSUE TITLE
Count for additional managers is overlapped/hidden by the status icon.

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#3196]
for additional managers we show `+ count` which getting overlapped by status icon of the project.

refer the image - 
<img width="194" alt="Screen Shot 2019-07-19 at 3 10 50 PM" src="https://user-images.githubusercontent.com/11248478/61526198-9585ae00-aa37-11e9-9c87-ab945406abef.png">

fyi @vikasrohit @maxceem

**Comment by vikasrohit** [comment]
Yep, it should be fixed as soon as we get rid of the white space created by `Join Project` button's column.
